src/components/PlaygroundComponent.css

```diff
@@ -104,10 +104,18 @@
   flex: 2;
 }
 
+.cfp-root:not(.cfp-open-preview) [data-idx="form-preview"] {
```

Comment:
Nitpick but maybe we can combine the two? Unless you want to keep it separate to have the same types be styled in groups. Although personally I think grouping around functionality makes more sense. If this were coded in JS, we'd probably have this switch represented in the same variable.

If your preference is against that, feel free to just merge 👍